src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/ProPlanControls.spec.jsx

```diff
@@ -0,0 +1,915 @@
+import { graphql, rest } from 'msw'
+import { setupServer } from 'msw/node'
+import { Suspense } from 'react'
+import { MemoryRouter, Route } from 'react-router-dom'
+
+import { TrialStatuses } from 'services/account'
+import { useAddNotification } from 'services/toastNotification'
+import { Plans } from 'shared/utils/billing'
+
+import ProPlanControls from './ProPlanControls'
+
+jest.mock('services/toastNotification')
+jest.mock('@stripe/react-stripe-js')
+jest.mock('./BillingControls', () => () => 'Billing Controls')
+jest.mock('./TotalBanner', () => () => 'Total Banner')
+jest.mock('./UserCount', () => () => 'User Count')
+
+const basicPlan = {
+  marketingName: 'Basic',
+  value: 'users-basic',
+  billingRate: null,
+  baseUnitPrice: 0,
+  benefits: [
+    'Up to 1 user',
+    'Unlimited public repositories',
+    'Unlimited private repositories',
+  ],
+  quantity: 1,
+  monthlyUploadLimit: 250,
+}
+
+const proPlanMonth = {
+  marketingName: 'Pro Team',
+  value: 'users-pr-inappm',
+  billingRate: 'monthly',
+  baseUnitPrice: 12,
+  benefits: [
+    'Configurable # of users',
+    'Unlimited public repositories',
+    'Unlimited private repositories',
+    'Priority Support',
+  ],
+  quantity: 10,
+  monthlyUploadLimit: null,
+}
+
+const proPlanYear = {
+  marketingName: 'Pro Team',
+  value: 'users-pr-inappy',
+  billingRate: 'annually',
+  baseUnitPrice: 10,
+  benefits: [
+    'Configurable # of users',
+    'Unlimited public repositories',
+    'Unlimited private repositories',
+    'Priority Support',
+  ],
+  monthlyUploadLimit: null,
+  quantity: 10,
+}
+
+const trialPlan = {
+  marketingName: 'Pro Trial Team',
+  value: 'users-trial',
+  billingRate: null,
+  baseUnitPrice: 12,
+  benefits: ['Configurable # of users', 'Unlimited repos'],
+  monthlyUploadLimit: null,
+}
+
+const allPlans = [
+  basicPlan,
+  proPlanMonth,
+  proPlanYear,
+  trialPlan,
+  {
+    marketingName: 'Pro Team',
+    value: 'users-enterprisem',
+    billingRate: 'monthly',
+    baseUnitPrice: 12,
+    benefits: [
+      'Configurable # of users',
+      'Unlimited public repositories',
+      'Unlimited private repositories',
+      'Priority Support',
+    ],
+    monthlyUploadLimit: null,
+  },
+  {
+    marketingName: 'Pro Team',
+    value: 'users-enterprisey',
+    billingRate: 'annually',
+    baseUnitPrice: 10,
+    benefits: [
+      'Configurable # of users',
+      'Unlimited public repositories',
+      'Unlimited private repositories',
+      'Priority Support',
+    ],
+    monthlyUploadLimit: null,
+  },
+  {
+    marketingName: 'Sentry',
+    value: 'users-sentrym',
+    billingRate: null,
+    baseUnitPrice: 0,
+    benefits: ['Includes 5 seats', 'Unlimited public repositories'],
+    monthlyUploadLimit: null,
+  },
+  {
+    marketingName: 'Sentry',
+    value: 'users-sentryy',
+    billingRate: null,
+    baseUnitPrice: 10,
+    benefits: ['Includes 5 seats', 'Unlimited private repositories'],
+    monthlyUploadLimit: null,
+  },
```

Comment:
Do we need to include these Sentry plans?

Reply:
True, the available_plans resolver wouldn't return these for this component